### ISSUE TITLE
Question about same_padding_conv

### ISSUE THREAD

**Issue by blackjack2015** [zjjMaiMai/TinyHITNet#17]
Hi, firstly many thanks for this helpful project. Easy to use and good performance.

I would like to know what is the difference between the default nn.conv2d and the following customized 'same_padding_conv'? Notice that 'same_padding_conv' also pads 'x' with zeros. Is it equivalent if one directly uses the default nn.conv2d?
https://github.com/zjjMaiMai/TinyHITNet/blob/6a78b3453bef927f66f04ca3f8949a15509cf3fc/models/hit_net_sf.py#L8

Thanks again and looking forward to your reply!

**Comment by zjjMaiMai** [comment]
> I would like to know what is the difference between the default nn.conv2d and the following customized 'same_padding_conv'? Notice that 'same_padding_conv' also pads 'x' with zeros. Is it equivalent if one directly uses the default nn.conv2d?

i just want to numeric align to origin tensorflow result, so we can transform tensorflow checkpoints to pytorch ckpt.
In some case, symmetric padding is not possible and nn.conv2d padding only one side. but im not sure nn.conv2d and tf.conv2d are padding same side or get same result. 

you can open a merge request if same_padding_conv and nn.conv2d give same result.
